README.md

```diff
@@ -20,349 +25,29 @@ Hyperledger Burrow is a permissioned blockchain node that executes smart contrac
-    function: getBytes
-
-- name: assertStorageBytes
-  assert:
-    key: $queryStorageBytes
-    relation: eq
-    val: $setStorageBaseBytes
-
-- name: setStorageBaseString
-  set:
-    val: nakaburrow
-
-- name: setStorageString
-  call:
-    destination: $deployStorageK
-    function: setString
-    data: [$setStorageBaseString]
-
-- name: queryStorageString
-  query-contract:
-    destination: $deployStorageK
-    function: getString
-
-- name: assertStorageString
-  assert:
-    key: $queryStorageString
-    relation: eq
-    val: $setStorageBaseString
-```
-
-while our Solidity contract (`storage.sol`) looks like:
-
-```
-pragma solidity >=0.0.0;
-
-contract SimpleStorage {
-  bool storedBool;
-  bool storedBool2;
-  int storedInt;
-  uint storedUint;
-  address storedAddress;
-  bytes32 storedBytes;
-  string storedString;
-
-  function setBool(bool x) {
-    storedBool = x;
-  }
-
-  function getBool() constant returns (bool retBool) {
-    return storedBool;
-  }
-
-  function setBool2(bool x) {
-    storedBool2 = x;
-  }
-
-  function getBool2() constant returns (bool retBool) {
-    return storedBool2;
-  }
-
-  function setInt(int x) {
-    storedInt = x;
-  }
-
-  function getInt() constant returns (int retInt) {
-    return storedInt;
-  }
-
-  function setUint(uint x) {
-    storedUint = x;
-  }
-
-  function getUint() constant returns (uint retUint) {
-    return storedUint;
-  }
-
-  function setAddress(address x) {
-    storedAddress = x;
-  }
-
-  function getAddress() constant returns (address retAddress) {
-    return storedAddress;
-  }
-
-  function setBytes(bytes32 x) {
-    storedBytes = x;
-  }
-
-  function getBytes() constant returns (bytes32 retBytes) {
-    return storedBytes;
-  }
-
-  function setString(string x) {
-    storedString = x;
-  }
-
-  function getString() constant returns (string retString) {
-    return storedString;
-  }
-}
-```
-
-Both files (`deploy.yaml` & `storage.sol`) should be in the same directory with no other yaml or sol files.
-
-From inside that directory, we are ready to deploy.
-
-```
-burrow deploy --address=F71831847564B7008AD30DD56336D9C42787CF63
-```
+## Installation
 
-where you should replace the `--address` field with the `ValidatorAddress` at the top of your `burrow.toml`.
+See the [install instructions](docs/INSTALL.md).
 
-That's it! You've succesfully deployed (and tested) a Soldity contract to a Burrow node.
+## Quick Start
+1. [Single full node](docs/quickstart/single-full-node.md)
+1. [Send transactions](docs/quickstart/send-transactions.md)
+1. [Deploy contracts](docs/quickstart/deploy-contracts.md)
+1. [Multiple validators](docs/quickstart/multiple-validators.md)
+1. [Seed nodes](docs/quickstart/seed-nodes.md)
 
-Note - that to redeploy the burrow chain later, you will need the same genesis-spec.json and burrow.toml files - so keep hold of them!
+## Project documentation
+Burrow documentation is available in the [docs](docs/README.md) directory.
```

Comment:
```md
Burrow getting started documentation is available in the [docs](docs/README.md) directory and in [GoDocs](https://godoc.org/github.com/hyperledger/burrow).
```

Reply:
Drop the 'in'.

It might be nice to include a table of contents into the docs directory. Bullets would be fine with a link and maybe brief strapline. e.g.

```markdown
- [Sending transactions](docs/quickstart/send-transactions.md) - how to communicate with your Burrow chain
```

Admittedly it is another thing to maintain but it probably provides less of a click-it-and-see experience